src/current/v24.1/cutover-replication.md

```diff
@@ -154,16 +159,129 @@ At this point, the primary and standby clusters are entirely independent. You wi
+    {{site.data.alerts.end}}
 
-For example, if you had [set up physical cluster replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}) between a primary and standby cluster and then cut over to the standby, the workflow to cut back to the original primary cluster would be as follows:
+1. Check the status of the virtual cluster on **A**:
 
-- Original primary cluster = Cluster A
-- Original standby cluster = Cluster B
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER {vc_a};
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+     id |  name  |     data_state     | service_mode
+    ----+--------+--------------------+---------------
+      1 | system | ready              | shared
+      3 | {vc_a} | replicating        | none
+      4 | test   | replicating        | none
+      (2 rows)
+    ~~~
+
+1. Run the following from **cluster A** to start the cutover:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} COMPLETE REPLICATION TO LATEST;
+    ~~~
+
+    The `cutover_time` is the timestamp at which the replicated data is consistent. The cluster will revert any data above this timestamp:
+
+    ~~~
+               cutover_time
+    ----------------------------------
+      1714497890000000000.0000000000
+    (1 row)
+    ~~~
+
+1. From **cluster A**, bring the virtual cluster online with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} START SERVICE SHARED;
+    ~~~
+
+1. To make **cluster A's** virtual cluster the default for connection strings, set the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING server.controller.default_target_cluster='{vc_a}';
+    ~~~
+
+At this point, **cluster A** is once again the primary and **cluster B** is once again the standby. The clusters are entirely independent. You will need to use your own network load balancers, DNS servers, or other network configuration to direct application traffic to the primary (**cluster A**). To enable physical cluster replication again, from the primary to the standby (or a completely different cluster), refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+
+### Cut back to a different cluster
+
+To create a completely new virtual cluster in the original primary cluster, or another physical cluster, use the [`CREATE VIRTUAL CLUSTER`]({% link {{ page.version.version }}/create-virtual-cluster.md %}) syntax:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE VIRTUAL CLUSTER new_vc FROM REPLICATION OF promoted_standby_vc ON connection_string_standby;
+~~~
```

Comment:
I don't know if we really need/want this section given that it is just starting a new replication stream. Open to removing and maybe just adding a note.

Reply:
yeah, I don't really see what this is adding